packaging/nnstreamer.spec

```diff
@@ -187,7 +187,7 @@ install build/gst/tensor_filter/*.a %{buildroot}%{_libdir}/
 %defattr(-,root,root,-)
 %license LICENSE
 %{_libdir}/*.so
-/usr/lib/nnstreamer/bin/*
+%{_bindir}/*
```

Comment:
Please keep the executables paths as /usr/lib/nnstreamer/bin/

Reply:
@myungjoo added a definition for examples-install dir. please review the changes again.